test/Microsoft.ML.CodeGenerator.Tests/Microsoft.ML.CodeGenerator.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="ApprovalTests" Version="3.0.19" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
You don't need these. They come automatically from:

[https://github.com/dotnet/machinelearning/blob/34b5e55ea143f7fc3a4acb33b9368f633819a845/test/Directory.Build.props#L29-L33](https://github.com/dotnet/machinelearning/blob/34b5e55ea143f7fc3a4acb33b9368f633819a845/test/Directory.Build.props#L29-L33)